src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -165,7 +166,7 @@ export const LocatorEditDialog: React.FC<Props> = ({
           locatorType,
           element_id,
           library,
-          message,
+          message: validationMessage,
           isCustomName: isEditedName,
         })
```

Comment:
correct me if I wrong, but I see that you these two actions are called with the same value. could you, please, refactor it not to duplicate the code?